package-lock.json

```diff
@@ -0,0 +1,12777 @@
+{
```

Comment:
There are both package-lock.json and yarn.lock here. Which one we are using? The other should probably be removed. At least, it is possible that dependabot will read the wrong file and make false positive/negative alerts.

Reply:
This slipped through in PR#10, I'll create a new PR for a fix.

Reply:
fix is now merged and appears in this PR too